src/nt.rs

```diff
@@ -139,6 +149,18 @@ impl NetworkTables<Server> {
             .unwrap()
             .add_server_callback(callback_type, action);
     }
+
+    pub fn create_rpc(
+        &mut self,
+        data: EntryData,
+        callback: impl Fn(Vec<u8>) -> Vec<u8> + Send + Sync + RefUnwindSafe + 'static,
+    ) {
+        self.state.lock().unwrap().create_rpc(data, callback);
+    }
+
+    pub fn call_rpc(&self, id: u16, parameter: Vec<u8>) -> Vec<u8> {
```

Comment:
Servers cannot initiate RPC Execute messages according to the spec

Reply:
fixed